.github/CODEOWNERS

```diff
@@ -677,6 +677,9 @@
 # PRLabel: %Mixed Reality
 /sdk/objectanchors/                                                @crtreasu @rgarcia @JoshLove-msft
 
+# PRLabel: %Remote Rendering
+/sdk/remoterendering/                                              @FlorianBorn71 @MichaelZp0 @ChristopherManthei
```

Comment:
```suggestion
# PRLabel: %Remote Rendering
/sdk/remoterendering/                                              @FlorianBorn71 @MichaelZp0 @ChristopherManthei

#ServiceLabel: %Remote Rendering
#ServiceOwners:                                                    @FlorianBorn71 @MichaelZp0 @ChristopherManthei
```

The original form would register reviewers for PRs only.   We'll also need to set up the block registering for support issues.